### PR TITLE
RDBS-7263 Additional logs

### PR DESCRIPTION
**Pull request by haludi**
Do Not Merge!!!
This is for Tzunami to debug their issue 
https://issues.hibernatingrhinos.com/issue/RDBS-7263

### CODE REVIEW

File: src/Raven.Server/Documents/DocumentsStorage.cs [hunk 1]
```diff
@@ -1798,6 +1798,9 @@ public void EnsureLastEtagIsPersisted(DocumentsOperationContext context, long do
                     tvb.Add(cv.Content.Ptr, cv.Size);
                     tvb.Add(lastModifiedTicks);
                     table.Insert(tvb);
+
+                    if (collectionName.Name.Equals("executiontasks", StringComparison.OrdinalIgnoreCase))
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/DocumentsStorage.cs:1802]: missing `_logger.IsOperationEnabled`

Comment (grisha-kotler) [inline src/Raven.Server/Documents/DocumentsStorage.cs]: missing `_logger.IsOperationEnabled`

File: src/Raven.Server/Documents/Indexes/Workers/CleanupDocuments.cs [hunk 2]
```diff
@@ -48,13 +48,23 @@ public class CleanupDocuments : IIndexingWork
 
             foreach (var collection in _index.Collections)
             {
+                var shouldExtraLogs = collection.Equals("executiontasks", StringComparison.OrdinalIgnoreCase);
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/Indexes/Workers/CleanupDocuments.cs]: missing `_logger.IsOperationEnabled`

File: src/Raven.Server/Documents/Indexes/Workers/CleanupDocuments.cs [hunk 3]
```diff
@@ -118,7 +136,15 @@ public class CleanupDocuments : IIndexingWork
                         continue;
 
                     if (_logger.IsInfoEnabled)
-                        _logger.Info($"Executing cleanup for '{_index} ({_index.Name})'. Processed {count} tombstones in '{collection}' collection in {collectionStats.Duration.TotalMilliseconds:#,#;;0} ms.");
+                    {
+                        _logger.Info(
+                            $"Executing cleanup for '{_index} ({_index.Name})'. Processed {count} tombstones in '{collection}' collection in {collectionStats.Duration.TotalMilliseconds:#,#;;0} ms.");
+                    }
+                    else if (_logger.IsInfoEnabled)
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/Indexes/Workers/CleanupDocuments.cs]:
```suggestion
                    else if (_logger.IsOpeartionEnabled)
```

File: src/Raven.Server/Documents/TombstoneCleaner.cs [hunk 4]
```diff
@@ -134,9 +135,17 @@ internal async Task ExecuteCleanup(long? numberOfTombstonesToDeleteInBatch = nul
                         }
 
                         if (tombstones.TryGetValue(tombstone.Key, out var item) == false)
+                        {
+                            if (tombstone.Key.Equals("executiontasks", StringComparison.OrdinalIgnoreCase))
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/TombstoneCleaner.cs]: missing `_logger.IsOperationEnabled`

File: src/Raven.Server/Documents/TombstoneCleaner.cs [hunk 5]
```diff
@@ -134,9 +135,17 @@ internal async Task ExecuteCleanup(long? numberOfTombstonesToDeleteInBatch = nul
...
+                                _logger.Operations($"{subscription}, {tombstone.Key}, {subscription.TombstoneCleanerIdentifier}, {tombstone.Value}");
                             tombstones[tombstone.Key] = (subscription.TombstoneCleanerIdentifier, tombstone.Value);
+                        }
                         else if (tombstone.Value < item.Value)
+                        {
+                            if (tombstone.Key.Equals("executiontasks", StringComparison.OrdinalIgnoreCase))
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/TombstoneCleaner.cs]: missing `_logger.IsOperationEnabled`

File: src/Raven.Server/Documents/TombstoneCleaner.cs [hunk 6]
```diff
@@ -186,6 +195,8 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                     try
                     {
                         var numberOfEntriesDeleted = _database.DocumentsStorage.DeleteTombstonesBefore(context, tombstone.Key, minTombstoneValue, numberOfTombstonesToDeleteInBatch);
+                        if (tombstone.Key.Equals("executiontasks", StringComparison.OrdinalIgnoreCase))
```
Comment (grisha-kotler) [inline src/Raven.Server/Documents/TombstoneCleaner.cs]: missing `_logger.IsOperationEnabled`

**Review by grisha-kotler — CHANGES_REQUESTED** [review]

**Comment by ppekrol** [comment]
Reopen when needed.

**Pull request closed without merging**